src/screens/SurveyDetail/index.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect, useContext } from 'react';
+
+import { useParams, Redirect } from 'react-router-dom';
+
+import Loader from 'components/Loader';
+import PageBackground from 'components/PageBackground';
+import { AuthContext } from 'contexts/auth';
+import { DetailsProvider } from 'contexts/details';
+import Detail from 'screens/SurveyDetail/detail';
+
+import Api from '../../utils/Api';
```

Comment:
@rafayet-monon Can we make some alias paths for this thing `../../utils` to `Utils/...` in the configuration?
It's ok if we can't make it 😅